scripts/globals/effects/mounted.lua

```diff
@@ -17,6 +17,15 @@ effect_object.onEffectGain = function(target, effect)
         target:ChangeMusic(4, 84)
         target:setAnimation(xi.anim.MOUNT)
     end
+
+    -- For PM3-3 The Road Forks.  This value will be checked periodically, and break
+    -- the Mimeo Jewel should a player be mounted, zone, or disconnect.
+    if
```

Comment:
(Optional) This might be a good opportunity to put in `xi.player.onPlayerMounted` and move this check in to there

Reply:
I wonder, do we want to keep this in a global scope, or implement it in a specific zone?

Reply:
Done